packages/components/src/pages/Chat.vue

```diff
@@ -0,0 +1,182 @@
+<template>
+  <div class="chat-container">
+    <div class="chat">
+      <small class="clear" @click="clear" v-if="isChatting">New chat</small>
+      <v-user-message
+        v-for="(message, i) in messages"
+        :key="i"
+        :message="message.message"
+        :is-user="message.isUser"
+      />
+      <v-chat-input
+        @send="onSend"
+        :placeholder="inputPlaceholder"
+        :class="inputClasses"
+        ref="input"
+      />
+      <v-suggestion-grid @suggest="onSuggestion" v-if="!isChatting" />
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+//@ts-nocheck
+import VUserMessage from '@/components/chat/UserMessage.vue';
+import VChatInput from '@/components/chat/ChatInput.vue';
+import VSuggestionGrid from '@/components/chat/SuggestionGrid.vue';
+import { AI, setConfiguration, DefaultApiURL } from '@appland/client';
+
+type Message = {
+  id?: string;
+  isUser: boolean;
+  message: string;
+  attachments?: string[];
+};
+
+export default {
+  name: 'v-chat',
+  components: {
+    VUserMessage,
+    VChatInput,
+    VSuggestionGrid,
+  },
+  props: {
+    apiKey: {
+      type: String,
+    },
+    apiUrl: {
+      type: String,
+      default: DefaultApiURL,
+    },
+  },
+  watch: {
+    apiKey() {
+      this.updateConfiguration();
+    },
+  },
+  data() {
+    return {
+      messages: [] as Message[],
+      threadId: undefined as string | undefined,
+    };
+  },
+  computed: {
+    inputPlaceholder() {
+      return 'Where do you want to go?';
```

Comment:
When I see this prompt in the demos, it's not quite landing for me; a little too abstract.